hack/preview.sh

```diff
@@ -39,6 +39,26 @@ if [ -n "$MY_GITHUB_ORG" ]; then
     $ROOT/hack/util-set-github-org $MY_GITHUB_ORG
 fi
 
+echo "start spi config"
+CLUSTER_URL_HOST=$(oc whoami --kubeconfig ${CLUSTER_KUBECONFIG} --show-console | sed 's|https://console-openshift-console.apps.||')
+if ! oc get namespace spi-system --kubeconfig ${KCP_KUBECONFIG} &>/dev/null; then
+  oc create namespace spi-system --kubeconfig ${KCP_KUBECONFIG}
+  oc create route edge -n spi-system --service spi-oauth-service --port 8000 spi-oauth --kubeconfig ${KCP_KUBECONFIG}
+fi
+export SPI_BASE_URL=https://$(kubectl --kubeconfig ${KCP_KUBECONFIG} get route/spi-oauth -n spi-system -o jsonpath='{.status.ingress[0].host}')
```

Comment:
We are already using two workspaces:
* compute
* redhat-appstudio

and if you run the preview.sh script on its own (not sure if this could ever happen though) then you don't know in which context (in which workspace) the kubeconfig is right now. 
It's very similar to namespaces - you specify in which namespace the call has to happen. We should make sure the same also for workspaces.